modules/storage-api/src/test/java/org/apache/ignite/internal/storage/AbstractPartitionStorageTest.java

```diff
@@ -168,13 +168,13 @@ public void scanFiltered() throws Exception {
         storage.write(dataRow1);
         storage.write(dataRow2);
 
-        List<DataRow> list = toList(storage.scan(key -> key.keyBytes()[3] == '1'));
+        List<DataRow> list = toList(storage.scan(key -> key.key().get(key.key().limit() - 1) == '1'));
```

Comment:
Can use `key.key().get(3)` ?

Reply:
This will fail in case DataRow has IEP-54 tuple inside.